.travis.yml

```diff
@@ -243,7 +243,7 @@ before_install:
       PY_CMD=python$PYTHON
       if [ "$TRAVIS_OS_NAME" = "osx" ]; then
         if [ "$PY" = "3" ]; then
-          brew update && brew unlink python@2 && brew upgrade python
+          brew update && brew unlink python@2 && brew upgrade python || brew install python
```

Comment:
This line is wrong. If `brew update` fails, this will still try to execute `brew install python`. Put the last two commands in parenthesis.

Reply:
Alright. That won't matter on these silly failures, though?

Reply:
Fixed.